pkg/schemas/management.cattle.io/v3/schema.go

```diff
@@ -326,11 +327,11 @@ func authzTypes(schemas *types.Schemas) *types.Schemas {
 		}).
 		MustImport(&Version, v3.GlobalRoleBinding{}).
 		MustImport(&Version, v3.RoleTemplate{}).
-		MustImport(&Version, v3.PodSecurityPolicyTemplate{}).
-		MustImportAndCustomize(&Version, v3.PodSecurityPolicyTemplateProjectBinding{}, func(schema *types.Schema) {
-			schema.CollectionMethods = []string{http.MethodGet, http.MethodPost}
-			schema.ResourceMethods = []string{}
-		}).
+		// MustImport(&Version, v3.PodSecurityPolicyTemplate{}).
```

Comment:
qq why are we commenting these here and in lines 305 - 307?

Reply:
I think it's a great point, however, the v3 management client needs to be updated - there's another ticket for it, as it's not in the scope of this PR, I think deleting the commented PSP templates would be safer after that, what do you think?

Reply:
sure, we can remove the code later in that ticket.